doc/cartridge_kubernetes_guide/index.rst

```diff
@@ -382,11 +382,11 @@ Comments provide a description of each parameter:
 
 With this configuration we will get the following:
 
-1. Tarantool Cartridge cluster called ``test-app``.
-2. Two replica sets in the cluster: ``routers`` and ``storages``.
-3. One tarantool instance in the ``routers`` replica set.
-4. Two instances, master and replica, in the ``storages`` replica set.
-5. Each replica set performs the roles listed in the ``RolesToAssign`` parameter.
+*  A Tarantool Cartridge cluster called ``test-app``.
```

Comment:
List items here just describe a set of objects, they aren't steps of an instruction. Hence, we don't need numbers.